Source/WebKit/WebProcess/WebProcess.h

```diff
@@ -706,7 +706,7 @@ class WebProcess : public AuxiliaryProcess
 
     WeakHashMap<WebCore::UserGestureToken, uint64_t> m_userGestureTokens;
 
-#if PLATFORM(WAYLAND)
+#if PLATFORM(WAYLAND) || (PLATFORM(GTK) && USE(EGL))
```

Comment:
This is weird, WAYLAND platform is only defined by the GTK port and EGL is requirement of the wayland platform.

Reply:
The only bot affected by this error is `Ubuntu 20.04`. This bot builds with `no-experimental-features` and with system libraries (no `flatpak` neither `jhbuild minimal`). I think the build error might be related to that.

Reply:
The reason why the build is failing in the Ubuntu 20.04 is because this bot builds with the flag ```--cmakeargs=-DENABLE_WAYLAND_TARGET=OFF```. Since WAYLAND is not enabled the variable `m_displayForCompositing` is not defined. But https://github.com/webkit/webkit/compare/d2424c0fc67f%5E..d2424c0fc67f#diff-d4c35777d04d911675eb95adf2a7cc767a915c0ed07068fcd02a7318b51a0015R126 makes use of this variable if PLATFORM is GTK and EGL is used.

I think the fix is correct. If `m_displayForCompositing` can be used under the condition `PLATFORM(GTK) && USE(EGL)` the variable should also be defined under that condition and not rely only on whether WAYLAND is enabled.